CMakeLists.txt

```diff
@@ -211,6 +215,19 @@ if (MATX_EN_NVPL OR MATX_EN_X86_FFTW)
             target_link_libraries(matx INTERFACE ${FFTW_LIB} ${FFTWF_LIB} ${FFTW_OMP_LIB} ${FFTWF_OMP_LIB})
             target_compile_definitions(matx INTERFACE MATX_EN_X86_FFTW=1)
         endif()
+
+        # BLAS
```

Comment:
Should there be a check to make sure that they don't have multiple BLAS libraries enabled (including NVPL)?

Reply:
Yes, will add

Reply:
Added